### DISCUSSION TITLE
Clarify Configuration File Locations in CLI Messages for `gh act`

### DISCUSSION THREAD

**Discussion by algo7** [nektos/act#2249] [Ideas]
When running the `act` command for the first time via the GitHub CLI extension (`gh act`), users are informed with the message:

> Default image and other options can be changed manually in ~/.actrc (please refer to https://github.com/nektos/act#configuration for additional information about file structure).

This message suggests that the configuration file (`~/.actrc`) is the primary (or only) place to modify the default settings. However, the underlying code supports multiple locations for the configuration file, as outlined by the `configLocations` function in `root.go`:

```go
// Return locations where Act's config can be found in order: XDG spec, .actrc in HOME directory, .actrc in invocation directory
func configLocations() []string {
	configFileName := ".actrc"

	homePath := filepath.Join(UserHomeDir, configFileName)
	invocationPath := filepath.Join(".", configFileName)

	// Though named xdg, adrg's lib support macOS and Windows config paths as well
	// It also takes cares of creating the parent folder so we don't need to bother later
	specPath, err := xdg.ConfigFile("act/actrc")
	if err != nil {
		specPath = homePath
	}

	// This order should be enforced since the survey part relies on it
	return []string{specPath, homePath, invocationPath}
}
```

Given the reliance on the `xdg.ConfigFile` function, the initial message might be a bit misleading for people using `gh act`. The existing docs aren't so clear about it also. 

Maybe introduce a different message for people using different installation methods or update the docs to clarify the location of the config file?

**Comment by GovtGeek** [comment]
I needed to update from micro to the medium. This would have saved me about an hour of searching. My settings (Ubuntu) were saved in `~/.config/act/atcrc` instead of `~/.actrc`.
